lib/pbench/server/api/resources/datasets_inventory.py

```diff
@@ -71,8 +66,12 @@ def _get(self, params: ApiParams, req: Request, context: ApiContext) -> Response
         cache_m = CacheManager(self.config, current_app.logger)
         try:
             file_info = cache_m.get_inventory(dataset.resource_id, target)
-        except (TarballNotFound, CacheExtractBadPath) as e:
-            raise APIAbort(HTTPStatus.NOT_FOUND, str(e))
+        except CacheManagerError as e:
+            raise APIAbort(HTTPStatus.NOT_FOUND, str(e)) from e
+        except Exception as e:
+            raise APIInternalError(
+                f"Unexpected error extracting {target} from {dataset.name}"
+            ) from e
```

Comment:
There's no unit test coverage for these lines.  This is not a big deal...it's just that we have 100% coverage for `lib/pbench/server/api/resources/datasets_compare.py` and `lib/pbench/server/api/resources/datasets_visualize.py`....  😉

Reply:
I know. 🐑 I started picking apart the unit test code to create a case and convinced myself not to "waste" more time yesterday afternoon. I felt baaad about that, so I guess now I'll need to do something about it.